### PR TITLE
get project details from the project id

### PR DESCRIPTION
**Pull request by karthiknag1**
get project details using the project id passed to the API.

### CODE REVIEW

**Review by aruniverse — COMMENTED** [review]
could you type the response

you also need to handle the version bump

**Review by aruniverse — APPROVED** [review]

File: src/ProjectsClient.ts [hunk 1]
```diff
@@ -46,6 +46,22 @@ export class ProjectsAccessClient implements ProjectsAccess {
     return this.getByUrl(accessToken, url);
   }
 
+  public async getByProjectId(accessToken: AccessToken, projectId: string): Promise<Project> {
```
[the remark this answers is not in the document]
Comment (karthiknag1) [reply src/ProjectsClient.ts:49]: sure will do.

**Comment by calebmshafer** [comment]
@karthiknag1 do you need this PR anymore? If not, please close it and delete the branch.

**Comment by SpencerWBarnes** [comment]
Duplicate of #19

**Pull request closed without merging**